geth/Dockerfile

```
@@ -12,6 +12,8 @@ RUN make geth
 # -------------------------------------------------------------------------
 FROM ubuntu:jammy
 
+RUN apt-get update && apt-get install -y ca-certificates && rm -rf /var/lib/apt/lists/*
```

Thanks for contributing.
For better readability, could you please format this line using multi-line style? (+ `node/Dockerfile`)

```suggestion
RUN apt-get update && \
    apt-get install -y ca-certificates && \
    rm -rf /var/lib/apt/lists/*
```